_posts/events/2018-12-13---azure-security-center-terraform-and-polyglot-communication-architectures.md

```diff
@@ -0,0 +1,58 @@
+Want to learn what Azure Security Center is about? 
+Interested in finding out whether Terraform is something for you? 
+Or want to design your next polyglot communication platform? 
+
+Then this evening is for you!
+
+## Agenda
+
+### Stay Safe with Azure Security Center
+Azure Security Center offers unified security management and advanced threat protection across hybrid cloud workloads.
+All of your Azure and on-prem workloads are continuously monitored to help find and remediate any vulnerabilities quickly. 
+Microsoft Intelligent Security Graph helps you to respond to continuously changing and improving cyber attacks. 
+With adaptive application controls and Just in time access you can protect your environment from various threats.
+ 
+This session will cover the security features baked into the Azure platform to help you keep your environment safe.
+
+<img src="/assets/media/speakers/els-putzeys.jpg" alt="Els Putzeys" align="left" height="100" width="100" style="margin-right: 20px;">**Speaker:** *Els Putzeys - With over 15 years of experience in the industry, Els Putzeys has knowledge on Windows Server, Exchange Server, SharePoint, Azure and Office 365. As a trainer, Els brings her real-world experience to the classroom. At U2U, she is responsible for the training material for all IT Pro courses.*
+
+### As an Azure user, should I switch to Terraform?
+One year ago, Microsoft reinforced their partnership with Hashicorp especially around Terraform, their resource management tool that gets a growing traction within the Cloud community. 
+Where are we one year later? How good is Terraform for Azure users? Should you adopt it? How does it compare to ARM templates?
+
+This talk will give you a glimpse of the Terraform ecosystem and how it integrates with Azure. 
+We will also talk about philosophy behind Terraform, the moving parts involved and the main usecases. 
+Terraform uses a declarative syntax to request resources and allocates them statefully, which is a game changer. Real life examples will be demonstrated live during the talk.
+
+This talk is for you if you want to know more about Terraform or if you still wonder if Terraform should be your next DevOps Azure move.
+
+<img src="/assets/media/speakers/olivier-dauby.jpg" alt="Olivier Dauby" align="left" height="100" width="100" style="margin-right: 20px;">**Speaker:** *Olivier Dauby - Olivier started his career more than 25 years ago in the sysadmin, network and datacenter worlds, then going up the virtualization ladder he is now evolving in cloud-native land. Working as a Cloud Infrastructure Architect at Stylelabs --the company behind the "m" Marketing Content Hub software-- one of his roles is to ensure streamlined deployment of "m" for their SaaS customers using Azure, DevOps CI/CD. Olivier is passionate about cloud services and is focussing on automation, rchestration and scalability techniques.*
+
+### Architecture of a Polyglot Communication Platform 
+
```

Comment:
Is this intentionally left blank without abstract?

Reply:
Yes. Gaston did not have an abstract … (yet)

Reply:
:+1: